lib/collections.js

```diff
@@ -0,0 +1,6 @@
+
+Games = new Mongo.Collection("games");
```

Comment:
Sure thing. 

The social media extensions build onto a "profile" attribute object within each user document.